nbs/models.tsmixerx.ipynb

```diff
@@ -658,12 +658,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.                    max_steps=200,
</pre><p>Again, I would use 500.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Changed to 500